appengine/README.md

```diff
@@ -0,0 +1,7 @@
+
+To deploy:
+  gcloud auth login ...
+  gcloud config set project mlab-sandbox
+  gcloud deploy queue.yaml
+
+See documentation online to update.
```

Comment:
Mentioning that task queues can be observed in cloud console would be a helpful pointer as well.